XCalendar.Forms/Converters/LocalizeDayOfWeekAndCharLimitConverter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+using XCalendar.Core.Extensions;
+using XCalendar.Forms.Resources;
+
+namespace XCalendar.Forms.Converters
+{
+    public class LocalizeDayOfWeekAndCharLimitConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (!(value is DayOfWeek dayOfWeek))
+            {
+                return string.Empty;
+            }
+
+            string resourceName = $"{dayOfWeek}";
+
+            string dayName = XCalendarResource.ResourceManager.GetString(resourceName, culture);
```

Comment:
Instead of managing translations ourselves, .NET provides a built-in method to do this:
https://learn.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo.getdayname?view=net-5.0

```suggestion
            string dayName = culture.DateTimeFormat.GetDayName(dayOfWeek);
```